### ISSUE TITLE
✨ Ensure breadcrumbs collapse on smaller resolutions

### ISSUE THREAD

**Issue by mitchdawson1982** [ministryofjustice/find-moj-data#996]
### Describe the feature request.

Based on feedback from accessibility assessment from Matt Proctor

_Element that needs some more work:
1. Breadcrumbs. When zoomed in, ideally breadcrumbs need to collapse otherwise they might take up too much space. There is a class you can add, "govuk-breadcrumbs--collapse-on-mobile" to achieve this. https://design-system.service.gov.uk/components/breadcrumbs/

The breadcrumb issue wasn't one that was highlighted in the review, and it is not technically an accessibility fail, more of a recommendation._

**Comment by murdo-moj** [comment]
https://github.com/ministryofjustice/find-moj-data/pull/1003
